package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "firebase": "^8.2.1",
-    "husky": "^4.3.6",
+    "firebase": "8.2.1",
+    "husky": "4.3.6",
     "lint-staged": "^10.5.3",
     "prettier": "^2.2.1",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
```

Comment:
eeep, super small! you'll want to keep the 'carets' `^` when you declare the versions of packages as the caret allows for all minor versions of that package (this is more important if you're building libraries but might be good to be aware of now). Here's a bit of reading if you have time:
- Semver (the way we version packages): https://semver.org/ (I honestly haven't read past the summary 👼 
- difference between tilde and caret in package.json files: https://medium.com/@Hardy2151/caret-and-tilde-in-package-json-57f1cbbe347b

Reply:
@gmzjuliana, regarding toMillis, it's part of the prototype of the property returned by Firebase in lastpurchasedDate.

Reply:
Those's dependencies that need to be fixed if we always want to work with the same version. For that reason, we removed that, but if you want we add them again.